spec/requests/atmosphere/api/user_keys_spec.rb

```diff
@@ -184,12 +181,8 @@
         }
         post api("/user_keys", user), invalid_user_key_req
         expect(response.status).to eq 422
-        expect(json_response['details']).to eq({
-          'public_key' => [
-            'bad type of key (only ssh-rsa is allowed)',
-            'is invalid'
-          ]
-        })
+        expect(json_response['details']['public_key']).
+          to include('bad type of key (only ssh-rsa is allowed)')
```

Comment:
I know the difference and that's why I am not sure sure if it's ok to replace one with another. I guess that test is supposed to check that response contains the given error message and optionally ensure hat other errors are not. My concern is about the optional part. If you don't care about checking whether other errors occurred then I'm ok with it.